- Sync -std= with devel/abseil to match ABI
- Avoid leaking abseil dependency to shared library consumers

--- CMakeLists.txt.orig	2020-07-15 02:27:54 UTC
+++ CMakeLists.txt
@@ -16,8 +16,8 @@
 cmake_minimum_required(VERSION 3.7.1 FATAL_ERROR)
 
 # libgav1 requires C++11.
-set(CMAKE_CXX_STANDARD 11)
-set(ABSL_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
+set(ABSL_CXX_STANDARD 17)
 
 project(libgav1 CXX)
 
@@ -107,8 +107,11 @@ if(NOT "${LIBGAV1_EXE_LINKER_FLAGS}" STREQUAL "")
   separate_arguments(LIBGAV1_EXE_LINKER_FLAGS)
 endif()
 
+find_package(absl)
+if (NOT TARGET absl::base)
 add_subdirectory("${libgav1_root}/third_party/abseil-cpp"
                  "${libgav1_abseil_build}" EXCLUDE_FROM_ALL)
+endif()
 
 libgav1_reset_target_lists()
 libgav1_add_dsp_targets()
